dockers/tpu-tests/Dockerfile

```diff
@@ -29,6 +29,9 @@ RUN cd pytorch-lightning && \
     unzip -o legacy/checkpoints.zip -d legacy/ && \
     ls -l legacy/checkpoints/
 
+# todo: torchtext
```

Comment:
maybe?
```suggestion
# todo: unpin torchtext version
```

Reply:
turned to be unrelated and skip this edit